src/groupByKey.ts

```diff
@@ -2,6 +2,19 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: any[], key: keyof T): GroupsMap<T> {
```

Comment:
```suggestion
export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
```
У тебя есть тип для этого, не используй any